pkg/cmd/api/api.go

```diff
@@ -513,3 +522,11 @@ func parseErrorResponse(r io.Reader, statusCode int) (io.Reader, string, error)
 
 	return bodyCopy, "", nil
 }
+
+func previewNamesToMIMETypes(names []string) string {
+	types := []string{fmt.Sprintf("application/vnd.github.%s-preview+json", names[0])}
```

Comment:
What is the reason why `+json` is only appended to the first preview header? I was unable to find anything in the media types docs.

Reply:
Good catch! `Accept` headers can list multiple MIME types, and is sufficient that only one of them specifies the json format, while the others are merely to activate additional previews and need not repeat `+json` any further.